### ISSUE TITLE
Option to exclude starred repositories

### ISSUE THREAD

**Issue by tomfitzhenry** [joeyh/github-backup#44]
I star a lot of repositories. Far more than I want to back up.

It would be nice to have `--no-starred`, in the same vain as `--no-forks`.

**Comment by ypid** [comment]
I agree. It would also be nice to have a switch `--only-starred`. This way one can backup own repos with forks and starred once with `--no-forks`.

**Comment by elsehow** [comment]
I would like this very much. I star a lot of things.
